app/models/user.rb

```diff
@@ -4,8 +4,19 @@
 class User < ApplicationRecord
   has_many :articles, dependent: :destroy
   has_many :comments, dependent: :destroy
+  has_many :followers, dependent: :destroy
+  has_many :follow_to,
+           class_name: :Follower,
+           foreign_key: :follower_id,
+           dependent: :destroy
 
   has_secure_password
   validates :email, presence: true, uniqueness: true
   validates :username, presence: true, uniqueness: true
+
+  def follow_user?(user_id)
```

Comment:
This method returns a boolean if the user is the same, and an object if is not. The return type should be the same.

Reply:
```
 def follow_user?(user_id)
    following = follow_to.find_by(user_id: user_id)
 end
```
I refactor the conditional in view and the method